primefaces-showcase/src/main/webapp/ui/input/oneRadio.xhtml

```diff
@@ -34,8 +34,8 @@
     <ui:define name="implementation">
         <div class="card">
             <h:form>
-                <h5 class="mt-0">Basic</h5>
-                <p:selectOneRadio id="console" value="#{radioView.console}" unselectable="true">
+                <h5 class="p-mt-0">Basic</h5>
```

Comment:
This seems to be wrong (probably due to a conflict)

Reply:
If you want you can edit this PR and fix it. Yeah it was a conflict I was trying to resolve

Reply:
Will have a look on my laptop